### PR TITLE
Added plugin check interval option

### PR DESCRIPTION
**Pull request by karupanerura**
I suggest to add `check_interval` option to checker plugin configuration.
I have to run a too heavy checker script...

### CODE REVIEW

**Comment by Songmu** [comment]
It's a nice pull request!

File: checks/checker.go [hunk 1]
```diff
@@ -94,5 +96,8 @@ func (c Checker) Check() (*Report, error) {
 // Interval is the interval where the command is invoked.
 // (Will be configurable in the future)
 func (c Checker) Interval() time.Duration {
```
Comment (Songmu) [inline checks/checker.go]: limit the range of return values.
It should be between 1 to 60 minutes.

If the configured value is out of range, 
truncate it to minimum or maximum value in the range
with warning logs.

File: config/config.go [hunk 2]
```diff
@@ -68,6 +68,7 @@ type PluginConfigs map[string]PluginConfig
 type PluginConfig struct {
 	Command              string
 	NotificationInterval *int32  `toml:"notification_interval"`
+	ExecutionInterval    *int32  `toml:"execution_interval"`
```
Comment (Songmu) [inline config/config.go]: ExecutionInterval doesn't work in metric plugins.
It's OK. I think it is preferred behaviour.
Please comment it.

**Comment by Songmu** [comment]
I pointed out the two points. Please fix them.
Thanks!

**Comment by karupanerura** [comment]
Thank you for your review. I'll fix it in near days.

**Comment by Songmu** [comment]
OK. Please `gofmt` at last!

**Comment by karupanerura** [comment]
oops! thank you for your notice.

**Comment by karupanerura** [comment]
done!

File: checks/checker.go [hunk 3]
```diff
@@ -94,5 +96,14 @@ func (c Checker) Check() (*Report, error) {
...
-	return 1 * time.Minute
+	if c.Config.ExecutionInterval != nil {
+		interval := time.Duration(*c.Config.ExecutionInterval) * time.Minute
+		if interval < 1*time.Second {
+			interval = 1 * time.Second
```
Comment (itchyny) [inline checks/checker.go]: s/Second/Minute/

**Comment by karupanerura** [comment]
🙇

File: checks/checker.go (hunk 3 above)
Comment (karupanerura) [inline checks/checker.go]: 🙇

File: config/config.go [hunk 4]
```diff
@@ -64,10 +64,11 @@ type Config struct {
 type PluginConfigs map[string]PluginConfig
 
 // PluginConfig represents a section of [plugin.*].
-// `MaxCheckAttempts` option is used with check monitoring plugins. Custom metrics plugins ignore this option.
+// `MaxCheckAttempts`, `NotificationInterval`, and `ExecutionInterval` option are used with check monitoring plugins. Custom metrics plugins ignore this option.
```
Comment (itchyny) [inline config/config.go]: `options are` ... `ignore these options`

Comment (karupanerura) [inline config/config.go]: oops! 🙇

**Comment by itchyny** [comment]
Looks Good to Me!

**Pull request merged**